_pages/how-we-work/equipment.md

```diff
@@ -17,6 +17,10 @@ GSA processes new badges every Wednesday. When your badge is ready, you’ll rec
 
 Badges are shipped to your local GSA office. You will receive an email once your badge has been shipped. At that point, you can make an appointment to pick it up. Your badge may be available during your orientation -- send an email to [hspd12.security@gsa.gov](mailto:hspd12.security@gsa.gov) to save yourself a trip. You may also ask your orientation leader to show you where to pick it up.
 
+## Business cards
+
+If you need business cards for a specific meeting with partners, you can request [GSA business cards](https://requests.18f.gov/gsa18f/procurements/new), which are free and don't reflect TTS or 18F branding. Upon receiving a request, a member of Team Ops will log into Envision Business Cards, then reach out to you to retrieve the information you want on the card.
+
```

Comment:
@fureigh Should we be clearer somehow that cards aren't really part of normal business and should only be ordered if there is a specific need?

Reply:
@tbaxter-18f If that isn't sufficiently clear, yes! :) How about this?

```suggestion
To make best use of our resources, we don't generally use business cards. If you need business cards for a specific meeting with partners, though, you can request [GSA business cards](https://requests.18f.gov/gsa18f/procurements/new) [...]
```

Reply:
(Trying to fix the syntax in that use of GitHub's `Suggested change` feature; hang on.)

Reply:
Looks like I've found a bug in the beta feature. 🙃

I've resubmitted the suggestion above. If it looks good to you, you can accept it directly; [here's how](https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change).

Reply:
I love it, but I still don't see a way to "commit suggestion"

Reply:
Yup, that's the secret: To commit, you have to be author or assigned.